src/coreclr/jit/hwintrinsic.h

```diff
@@ -534,21 +534,27 @@ struct HWIntrinsicInfo
 
             case NI_SSE41_Ceiling:
             case NI_SSE41_CeilingScalar:
+            case NI_AVX_Ceiling:
+            {
+                FALLTHROUGH;
```

Comment:
nit: remove  `{ FALLTHROUGH; }`

Reply:
I had it split "explicitly" here to help visualize the groupings between `Ceiling` vs `RoundToPositiveInfinity` (and likewise `Floor` vs `RoundToNegativeInfinity`), particularly since Ceiling/Floor don't have AVX512 equivalents.